src/components/doctors/DoctorDetails.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { NavLink, useParams } from 'react-router-dom';
+
+import {
+  selectDoctors,
+  selectDoctorsStatus,
+  selectDoctorsError,
+} from '../../redux/doctorsSlice';
+import {
+  selectAppointmentsStatus,
+  selectAppointmentsError,
+  createAppointment,
+} from '../../redux/appointmentsSlice';
+
+function DoctorDetails() {
```

Comment:
- [ ] If I go to a doctors detail page, and press the "reserve appointment" button:

![image](https://github.com/CaptainBawa/DocBook-Front-End/assets/57421823/e140837c-b97b-42e0-97a2-cf69abb588d4)

It will take me to the reservations form, but it will make a request to the server trying to create the appointment before even taking me to the form.  Of course, that request will fail:

![image](https://github.com/CaptainBawa/DocBook-Front-End/assets/57421823/00cb4287-3df3-48bc-898f-acfc14cdc526)

Kindly, fix this bug so your app works as intended. 👍